c-sharp-razor/Dockerfile

```diff
@@ -0,0 +1,38 @@
+# Base image for the runtime (using Alpine)
+FROM mcr.microsoft.com/dotnet/aspnet:8.0-alpine AS base
+WORKDIR /app
+EXPOSE 5000
+
+# Build the application (using Alpine SDK)
+FROM mcr.microsoft.com/dotnet/sdk:8.0-alpine AS build
+ARG BUILD_CONFIGURATION=Release
+WORKDIR /src
+
+# Copy project files
+COPY ["src/Minitwit.Web/Minitwit.Web.csproj", "src/Minitwit.Web/"]
+COPY ["src/Minitwit.Core/Minitwit.Core.csproj", "src/Minitwit.Core/"]
+COPY ["src/Minitwit.Infrastructure/Minitwit.Infrastructure.csproj", "src/Minitwit.Infrastructure/"]
+
+# Restore dependencies
+RUN dotnet restore "src/Minitwit.Web/Minitwit.Web.csproj"
+
+# Copy all source files and build
+COPY . .
+RUN dotnet build "src/Minitwit.Web/Minitwit.Web.csproj" -c $BUILD_CONFIGURATION -o /app/build
```

Comment:
Does it makes sense? Your a copying just a few things and then restoring dependencies and then you are copying everything... Correct me if I'm wrong but it wouldnt make sense just to copy everything -> Install dependencies >> build?

Reply:
Adressed comment